PROVISIONING.md

```diff
@@ -23,10 +25,10 @@ tbd... re-celling, faults...
 NVIDIA has a manual provisioning process based on a virtual *-over-PCIe device set, called [RSHIM](https://github.com/Mellanox/rshim). RSHIM creates, among other things, a virtual point-to-point ethernet device, and a virtual console device, between host and DPU. See also [usage](https://docs.nvidia.com/networking/display/BlueFieldDPUOSLatest/Deploying+DPU+OS+Using+BFB+from+Host) of RSHIM.
 Many customers are using this process to deploy their own OS image and initialize system configuration, since they trust the OS running on the x86 host.
 
-## USB/Virtual media Provisioning
+Andy: Do we need to cover manual methods in our scope? Many, many methods. 
 
+### USB/Virtual media Provisioning (this is typically a manual method, enabled by a one-to-one interaction with a BMC)
```

Comment:
Minor nit: Maybe move the wording in parenthesis into a sentence outside of the header.